lib/takeaway.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+  DISHES = {:burger => 5, :hotdog => 4, :fries => 3, :milkshake => 4, :soda => 3}
```

Comment:
Line is too long. [81/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.